xml/System.Text.Encodings.Web/HtmlEncoder.xml

```diff
@@ -82,10 +82,16 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="allowedRanges">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="allowedRanges">The set of characters that the encoder is allowed to not encode.</param>
+        <summary>Creates a new instance of the HtmlEncoder class specifying characters to encode.</summary>
```

Comment:
specifying --> that specifies

It appears, though, that the parameter is not characters to encode, but ranges the encoder is allowed to not encode. So perhaps it would be better to write "characters the encoder is allowed to not encode."

This comment also applies to the other encoding Default properties.